sympy/integrals/integrals.py

```diff
@@ -520,8 +521,14 @@ def try_meijerg(function, xab):
                         function = antideriv._eval_interval(x, a, b)
                         function = Poly(function, *gens)
                     elif isinstance(antideriv, Add):
```

Comment:
I think this alternative should be entirely removed. The whole ``antideriv`` could be passed to ``_eval_interval`` as is done after the next ``else:`` statement.